etc/DependencyInstaller.sh

```diff
@@ -213,15 +218,30 @@ _installOrTools() {
     if [[ ! -z "${PREFIX}" ]]; then mkdir -p "${PREFIX}"; fi
     cd "${baseDir}"
 
-    orToolsFile=or-tools_${arch}_${os}-${version}_cpp_v${orToolsVersionSmall}.tar.gz
-    wget https://github.com/google/or-tools/releases/download/v${orToolsVersionBig}/${orToolsFile}
     orToolsPath=${PREFIX:-"/opt/or-tools"}
-    if command -v brew &> /dev/null; then
-        orToolsPath="$(brew --prefix or-tools)"
+    if [ "$(uname -m)" == "aarch64" ]; then
+        LIST=($(sudo find / -type f -name "libortools.so*" 2>/dev/null))
```

Comment:
I would like to avoid using sudo in this script. One of the reasons to subdivide the install process for each OS and the common packages (i.e., those installed from source) is to support some users that do not have sudo access. If a function has a command that needs sudo it should be executed with the other `-dev` functions.